### PR TITLE
VMRay: Feature - Add get screenshots action

### PR DESCRIPTION
**Pull request by sw-vmray**
Please ensure your pull request (PR) adheres to the following guidelines:

- Please refer to our contributing documentation for any questions on submitting a pull request, link: [Contribution Guide](https://github.com/splunk-soar-connectors/.github/blob/main/.github/CONTRIBUTING.md)

#### Please check if your PR fulfills the following requirements:
- [x] Testing of all the changes has been performed (for bug fixes / features)
- [x] The manual_readme_content.md has been reviewed and added / updated if needed (for bug fixes / features)
- [x] Use the following format for the PR description: `<App Name>: <PR Type> - <PR Description>`
- [x] Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.
- [x] Verify all checks are passing.
- [x] Do NOT use the `next` branch of the forked repo. Create separate feature branch for raising the PR.
- [x] Do NOT submit updates to dependencies unless it fixes an issue.
#### Please check the type of change your PR introduces:
- [ ] New App
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation
- [ ] Other (please describe):

## Security Considerations (REQUIRED)
- [ ] If you are exposing any endpoints using a [REST handler](https://docs.splunk.com/Documentation/SOAR/current/DevelopApps/RESTHandlers),
  please document them in the `manual_readme_content.md`.
- [ ] If this is a new connector or you are adding new actions
    - [ ] Please document in the `manual_readme_content.md` all methods (eg, OAuth) used to authenticate
      with the service that the connector is integrating with.
    - [ ] If any actions are unable to run on SOAR Cloud, please document this in the `manual_readme_content.md`.
- [ ] Are you introducing any new cryptography modules? If yes, please elaborate their purpose:
- [ ] Are you are accessing the file system? If yes, please verify that you are only accessing paths returned through
the [Vault](https://docs.splunk.com/Documentation/SOARonprem/5.2.1/DevelopApps/AppDevAPIRef#Vault) API.
- [ ] Are you are marking code to be ignored by Semgrep with [`nosemgrep`](https://semgrep.dev/docs/ignoring-files-folders-code/#ignoring-code-through-nosemgrep)?
If yes, please provide justification in an additional comment next to the ignored code.

## Release Notes (REQUIRED)
Added action to get screenshots from a dynamic analysis.

## What is the current behavior? (OPTIONAL)
~~- Describe the current behavior that you are modifying.~~ 

## What is the new behavior? (OPTIONAL)
~~- Describe the behavior or changes that are being added by this PR.~~

## Other information (OPTIONAL)
~~- Any other information that is important to this PR such as screenshots of how the component looks before and after the change.~~

## Pay close attention to (OPTIONAL)
~~- Any specific code change or test case points which must be addressed/reviewed at the time of GA release.~~

## Screenshots (if relevant)

---
Thanks for contributing!

### CODE REVIEW

**Comment by spopat-crest** [comment]
Hello @sw-vmray, prodsec-scans pipeline is failing due to requests-2.25.0 module present in the requirements.txt, could you please provide me the write access for the branch or remove **requests** from the requirements file and verify if the connector is working fine.

**Comment by spopat-crest** [comment]
Hello @sw-vmray Could you please provide me the write access to the branch in order to fix prodsec-scans failures

**Comment by sw-vmray** [comment]
Hi @spopat-crest, sorry for the late answer.

The requests package is required. This is what we use to communicate with the VMRay Platform.

I gave you the write access to the fork.

**Comment by ishans-crest** [comment]
Hi @sw-vmray,
As stated by @spopat-crest Prodsec is encountering issues due to the inclusion of the `request` module of version `2.25.0` in the `requirements.txt` file. The Splunk SOAR platform already has the `request` library in its environment, so there is no need to explicitly specify it in the requirements.

File: release_notes/unreleased.md [hunk 1]
```diff
@@ -1 +1,2 @@
 **Unreleased**
+* Added the 'create meeting' action to get screenshots from a dynamic analysis
```
Comment (sw-vmray) [inline release_notes/unreleased.md]: Is `create meeting` a typo? Or a placeholder for something? Shouldn't it be `get screenshots`?

Comment (spopat-crest) [reply release_notes/unreleased.md]: yes it was a typo

**Review by achandya-crest — APPROVED** [review]

**Review by ishans-crest — APPROVED** [review]

**Pull request merged**